parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveType.java

```diff
@@ -459,6 +468,37 @@ public PrimitiveType(Repetition repetition, PrimitiveTypeName primitive,
     this.columnOrder = requireValidColumnOrder(columnOrder);
   }
 
+  public PrimitiveType(Repetition repetition, PrimitiveTypeName primitive,
```

Comment:
I think, in long term we do not want to expose the construction of types from the API. We would like the clients to use the builder instead. Therefore, I would suggest not adding public constructors if your code does not need it.

Reply:
Ah sorry, missed this one. Sure, I can make it package private